scripts/init

```diff
@@ -2,6 +2,20 @@ export DEVENVROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
 
 . ${DEVENVROOT}/scripts/env.d/vars
 
+#----------------------------------------------------------------------------------------------------
+#    make sure system make is installed and reachable 
+#----------------------------------------------------------------------------------------------------
+#  find make
+MAKE=$(which make 2>/dev/null)
+
+if (( $? == 1 )); then 
```

Comment:
I would like this to be fenced by an environment variable, like `$DEVENVVERBOSE` or `$DEVENVDEBUG`.  I think most of the time we would like `devenv` to be lean.